tests/lib/BinaryFinderTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace Test;
+
+use OC\BinaryFinder;
+use OC\Memcache\ArrayCache;
+use OCP\ICache;
+use OCP\ICacheFactory;
+use OCP\IConfig;
+
+class BinaryFinderTest extends TestCase {
+	private ICache $cache;
+	private ICacheFactory $cacheFactory;
+	private $oldEnv;
+
+	protected function setUp(): void {
+		$this->oldEnv = getenv("PATH");
```

Comment:
```suggestion
		$this->oldEnv = getenv('PATH');
```

To make the linter pass